app/models/importers/hmis_auto_migrate/local.rb

```diff
@@ -37,6 +39,7 @@ def import!
     end
 
     delegate :loader_log, to: :@importer
+    delegate :importer_log, to: :@importer
```

Comment:
small housekeeping item but it looks like we can use the `importer` accessor that was added
```suggestion
    delegate :importer_log, to: :importer
```